premailer/premailer.py

```diff
@@ -264,7 +268,10 @@ def transform(self, pretty_print=True, **kwargs):
 
         assert page is not None
 
-        head = get_or_create_head(tree)
+        if self.disable_leftover_css:
+            head = None
+        else:
+            head = get_or_create_head(tree)
```

Comment:
This might not be enough. You might also need to add `and not self.disable_leftover_css` [here](https://github.com/ionelmc/premailer/blob/master/premailer/premailer.py#L336) and [here](https://github.com/ionelmc/premailer/blob/master/premailer/premailer.py#L343) too.